src/main.ts

```diff
@@ -105,7 +105,17 @@ function resolveVersionInput(): string {
       );
     }
 
-    version = parseNodeVersionFile(fs.readFileSync(versionFilePath, 'utf8'));
```

Comment:
previously, it was assumed that _if_ a version-file was provided, it would always contain a node version.

This is not the case